docs/getting-started.md

```diff
@@ -26,8 +26,33 @@ Check out the [Local Environment](./local-environment.md) document.
 
 First of all, you need to make sure that all PHP and JavaScript dependencies are installed:
 
+PHP will be installed on Mac by default
+
+Install Composer, please use the following command or see [here](https://getcomposer.org/download/)
+
+```bash
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+php -r "if (hash_file('sha384', 'composer-setup.php') === 'c31c1e292ad7be5f49291169c0ac8f683499edddcfd4e42232982d0fd193004208a58ff6f353fde0012d35fdd72bc394') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+php composer-setup.php
+php -r "unlink('composer-setup.php');"
```

Comment:
No need to duplicate Composer's install instructions.

Reply:
Removed